pkg/operators/ebpf/maps.go

```diff
@@ -99,7 +101,14 @@ func (i *ebpfInstance) evaluateMapParams(paramValues api.ParamValues) error {
 	globalDuration := time.Duration(0)
 	globalCount := 0
 
-	durations, err := apihelpers.GetDurationValuesPerDataSource(paramValues[ParamMapIterInterval])
+	mapIterInterval := paramValues[ParamMapIterInterval]
```

Comment:
LGTM, but it's fixing a specific use case. This is probably happening in other situations we haven't hit yet, and final solution should look like what you mentioned in the PR's description.

I'm wondering why `GetGadgetInfo` is not called when using headless mode. It means that we're not validating at all the parameters and don't provide any feedback if they are wrong? We simply print `INFO[0000] installed as "XXX"` but gadget is not actually running?